### ISSUE TITLE
About warning for mtl_transformer_sp

### ISSUE THREAD

**Issue by DamonYangyang** [athena-team/athena-decoder#20]
when using tf.fuction, a warning will apear,the content is "the length of logits is shorter than taht of labels "  "WARNING:tensorflow:Model was constructed with shape (None, 512) for input Tensor("dense_5_input:0", shape=(None, 512), dtype=float32), but it was called on an input with incompatible shape (None, None, 512)"
